
path_source: param
path_param: block_6

use_gps_origin: True

